safe/definitions/styles.py

```diff
@@ -22,10 +22,10 @@
 # Map legend templates according to standards :
 # https://github.com/inasafe/inasafe/issues/3653#issuecomment-275011957
 # The space between count and exposure_unit will be automatically added if
-# the unit is not empty..
+# the unit is not empty.
 template_without_thresholds = u'{name} ({count}{exposure_unit})'
 template_with_maximum_thresholds = (
-    u'{name} < {max} {hazard_unit} ({count}{exposure_unit})')
+    u'{name} <= {max} {hazard_unit} ({count}{exposure_unit})')
 template_with_minimum_thresholds = (
     u'{name} > {min} {hazard_unit} ({count}{exposure_unit})')
```

Comment:
I wanted to change this one to : 
`u'{name} > {min} <= {max} {hazard_unit} ({count}{exposure_unit})')`
But I'm not convinced.

This is the label used in the legend.